src/blocks/TrustpilotBlock/TrustpilotBlock.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useRef } from 'react'
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { ContentWrapper } from 'components/blockHelpers'
+import { ContextContainer } from 'components/containers/ContextContainer'
+
+const TrustpilotWrapper = styled(ContentWrapper)`
+  && {
+    max-width: 1200px;
+  }
+`
```

Comment:
I'm wondering if this is really necessary? If you don't pass any particular props to `ContentWrapper` you'll get this exact `max-width`. And if you still want to add this styling to be sure it's always there, even if someone change or remove the `CONTENT_MAX_WIDTH_DEPRECATED` variable in `blockHelpers.tsx" it's enough with this I think:
```suggestion
const TrustpilotWrapper = styled(ContentWrapper)`
  max-width: 1200px;
`
```

Reply:
That is correct! But it also gives me a `maxWidth: 1500` that looks way off for this particular component. Since I don't want to remove `CONTENT_MAX_WIDTH_DEPRECATED` in this PR I need to override that with `&&` 
```
export const CONTENT_MAX_WIDTH_DEPRECATED = {
  maxWidth: 1200,
  [GIANT_BP_UP]: {
    maxWidth: 1500,
  },
}
```

Reply:
But I absolutely agree that the way we handle max-widths at the moment is not optimal 😄

Reply:
Ah, OK I see 👍